cli/main.rs

```diff
@@ -106,6 +106,7 @@ impl From<Flags> for PermissionsOptions {
 
 fn create_web_worker_callback(
   program_state: Arc<ProgramState>,
+  parent_thread_permissions: Permissions,
```

Comment:
This is not right, `create_web_worker_callback` is called at the start of the whole program which means it will include permissions from startup instead of current permissions parent thread has. This field should be added to the `args` below (`CreateWebWorkerArgs` struct)